server/paths/match.ts

```diff
@@ -32,10 +31,9 @@ export default {
     index: placementRequestsPath,
     show: placementRequestPath,
     create: newPlacementRequestPath,
-    bookingNotMade: { confirm: bookingNotMadePath.path('confirm'), create: bookingNotMadePath },
-    bookings: {
-      confirm: placementRequestBookingsPath.path('confirm'),
-      create: placementRequestBookingsPath,
+    bookingNotMade: {
```

Comment:
Could we maybe change to `placementNotMade` for consistency

Reply:
This could do with a small refactor -- the 'Unable to match' flow uses the old implementation, which seems to work fine, but could do with moving over to the newer `spaceBookingsController`!